docs/manual/cli.md

```diff
@@ -377,40 +340,50 @@ name by a number:
 
 ## Showing help about Redis commands
 
-Redis has a number of [commands](/commands) and sometimes, as you test things,
-you may not remember the exact order of arguments. `redis-cli` provides online
-help for most Redis commands, using the `help` command. The command can be used
+`redis-cli` provides online help for most Redis [commands](/commands), using the `HELP` command. The command can be used
 in two forms:
-
-* `help @<category>` shows all the commands about a given category. The
-categories are: `@generic`, `@list`, `@set`, `@sorted_set`, `@hash`,
-`@pubsub`, `@transactions`, `@connection`, `@server`, `@scripting`,
-`@hyperloglog`.
-* `help <commandname>` shows specific help for the command given as argument.
+* `HELP @<category>` shows all the commands about a given category. The
+categories are: 
+    - `@generic`
+    - `@string`
```

Comment:
There needs to be a blank line before the sublist here. Otherwise, it won't show up as a list (in the current website anyway).

Reply:
Added. Thanks, @zuiderkwast